homeassistant/components/camera/__init__.py

```diff
@@ -328,15 +328,20 @@ async def write_to_mjpeg_stream(img_bytes: bytes) -> None:
     return response
 
 
+async def init_camera_prefs(hass: HomeAssistant) -> CameraPreferences:
+    """Initialize camera preferences."""
+    prefs = CameraPreferences(hass)
+    await prefs.async_load()
+    hass.data[DATA_CAMERA_PREFS] = prefs
+    return prefs
```

Comment:
Why is this changed?

Reply:
Good spot - this was needed for a previous iteration. Doesn't need to be split out now.
Fixed in c9d4c90c